### ISSUE TITLE
How do you split the training set and test set

### ISSUE THREAD

**Issue by zhaojiachen1994** [okankop/Driver-Anomaly-Detection#3]
This is a great project. I learn a lot from it. However, I have a little question. How do you split the training set and the test set? I have seen the DAD class and DAD_Test class for loading the dataset, but it seems to load the same data for training and testing. Can you give me a hand about this?

**Comment by okankop** [comment]
Happy to hear that you find our project usefull.

Training and validation set are split by subject. Please check out the [main.py](https://github.com/okankop/Driver-Anomaly-Detection/blob/master/main.py), where for both sets DAD class is used, but the subset is set to ‘train’ and ‘validation’ accordingly.

DAD_Test class is created only to be used at [demo_live.py](https://github.com/okankop/Driver-Anomaly-Detection/blob/master/demo_live.py) to genrate consecutive video samples. In this class validation set is used.
